_html/templates/mail.html

```diff
@@ -0,0 +1,574 @@
+<!DOCTYPE html>
+<html>
+    <head lang="en">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <style type="text/css">
+            * {
+                margin: 0 auto;
+                padding: 0;
+                font-size: 16px;
+            }
+
+            table {
+                border-collapse: collapse;
+                border-spacing: 0;
+            }
+
+            @media only screen and (max-width: 480px) {
+                .content-wrapper,
+                .templateContainer {
+                    width: 100% !important;
+                }
+                
+                img {
+                    max-width: 100% !important;
+                }
+
+                .block {
+                    display: inline-block!important;
+                    width: 100%!important;
+                    text-align: center!important;
+                }
+
+                .mobile-reposition {
+                    text-align: center!important;
+                }
+
+                .mobile-exclude {
+                    display: none;
+                }
+            }
+
+        </style>
+    </head>
+    <body bgcolor="#ffffff" style="text-align: center; background-color: #ffffff;">
+        <center>
+            <table border="0" cellpadding="0" cellspacing="0" width="100%" bgcolor="#ffffff" class="content-wrapper">
+                <tr style="text-align: center;">
+                    <td>
+                        <table align="center" border="0" cellpadding="0" cellspacing="0" width="600px" class="templateContainer" bgcolor="#200F17">
+                            <tr>
+                                <td>
+                                    <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+                                        <tr>
+                                            <td>
+                                                <img src="http://frontend-news.dev.pitechplus.com/drupal_camp_mail/small-header.png" alt="First row background">
+                                            </td>
+                                        </tr>
+                                    </table>
+                                    <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+                                        <tr>
+                                            <td style="padding: 15px;">
+                                                <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+                                                    <tr>
+                                                        <td>
+                                                            <span></span>
```

Comment:
Is this needed? If not please remove it, otherwise ignore the comment.

Reply:
i use them as "spaces " between elements